chantal/build.py

```diff
@@ -46,13 +46,16 @@ def build_job(args):
         else:
             shallow = ""
 
-        run_command("git clone " + shallow +
-                    shlex.quote(args.clone_location) +
-                    " " + args.work_location, base_env)
+        run_command("git init " + args.work_location, base_env)
 
     os.chdir(args.work_location)
 
     if args.treeish:
```

Comment:
if we don't get a treeish we would not fetch any repo content - so the git init should be part of this condition, and in the else part we fall back to git clone